wcivf/settings/base.py

```diff
@@ -281,7 +281,7 @@ def get_ec2_ip():
 
 AKISMET_API_KEY = os.environ.get("AKISMET_API_KEY")
 
-AKISMET_BLOG_URL = CANONICAL_URL
+PYTHON_AKISMET_BLOG_URL = CANONICAL_URL
```

Comment:
You're still using `AKISMET_BLOG_URL`, so can you revert this line too?

Reply:
That's done now, thanks for catching.